index.html

```diff
@@ -14,10 +14,32 @@
     <header>
       <div class="header-nav-phone">
         <a href="/" class="header-logo"> Welcome </a>
-        <nav>
-          <button type="button" class="menu-btn">
-            <img src="./asset/Button.svg" alt="menu link" />
+
+        <button id="mobile-menu-button" type="button" class="menu-btn">
+          <img src="./asset/Button.svg" alt="menu link" />
+        </button>
+        <nav id="mobile-menu-hide" class="mobile-menu-hide">
+          <button type="button" id="btn-close-nav">
+            <img src="./asset/Icon-Cancel.svg" alt="cancel icon">
           </button>
+          <ul>
+            <li>
+              <a href="#myworks-id" class="link-portfolio">
+                Portfolio
+              </a>
+            </li>
+            <li>
+              <a href="#aboutme-id" class="link-about">
+                About
+              </a>
+            </li>
+            <li>
+              <a href="#footer-contact-id" class="link-contact">
+                Contact
+              </a>
+            </li>
```

Comment:
- [ ] Please make sure to implement the same functionality of the mobile menu nav links to the desktop nav links.
- [ ] Kindly make sure that the close button is visible on mobile view, (See below image)
![image](https://user-images.githubusercontent.com/62290545/167648931-920f1896-0322-4f6f-ba83-975ebbc07f57.png)